lib/rubocop/cop/lint/redundant_regexp_quantifiers.rb

```diff
@@ -5,6 +5,9 @@ module Cop
     module Lint
       # Checks for redundant quantifiers inside Regexp literals.
       #
+      # It is always allowed when interpolation is used in a regexp literal,
```

Comment:
I'd also add an example illustrating the use of interpolation.

Reply:
Sure! I added an example for it.